Source/AwsCommonRuntimeKit/io/Stream.swift

```diff
@@ -49,61 +45,18 @@ extension FileHandle: AwsStream {
     }
```

Comment:
above this, seekToEndOfFile() is used, but that's a deprecated function because it doesn't report errors, we should use seekToEnd()

likewise seek(toFileOffset) is deprecated and seek(toOffset) should be used so we don't crash the application on errors

which means the lenght() protocol function should be throwing

...

and now that I'm thinking about it since a failed seek can REALLY mess up the stream, maybe we should just have InputStreamCore store `self.valid = false` any time one of its protocol functions throw, and we can report that via `aws_input_stream_get_status()`

Reply:
Thanks, fixed.